src/server/plugins/router.ts

```diff
@@ -14,7 +14,9 @@ const privacyPolicies: Record<string, string> = {
   'register-a-boat-at-rye-harbour':
     'https://www.gov.uk/government/publications/rye-harbour-privacy-notice',
   'apply-for-an-annual-mooring-at-rye-harbour':
-    'https://www.gov.uk/government/publications/rye-harbour-privacy-notice'
+    'https://www.gov.uk/government/publications/rye-harbour-privacy-notice',
+  'request-a-form':
+    'https://defra.sharepoint.com/sites/Community4442/SitePages/Privacy-notice--Defra-Forms.aspx'
```

Comment:
Do SharePoint pages have short URLs that might be a little less fragile?

Reply:
Unfortunately not